openupgrade_scripts/scripts/analytic/18.0.1.1/post-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2025 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade, openupgrade_180
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env, "analytic", "18.0.1.1/noupdate_changes.xml")
```

Comment:
please, rebase and move everything to 18.0.1.2

Reply:
done